src/CommunityToolkit/Xamarin.CommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -154,20 +154,20 @@
     <Reference Include="PresentationCore" />
     <Reference Include="PresentationFramework" />
     <Reference Include="WindowsBase" />
-    <PackageReference Include="Xamarin.Forms.Platform.WPF" Version="5.0.0.2125" />
+    <PackageReference Include="Xamarin.Forms.Platform.WPF" Version="5.0.0.2291" />
     <PackageReference Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
   </ItemGroup>
 
   <ItemGroup Condition=" $(TargetFramework.StartsWith('netcoreapp')) AND '$(OS)' == 'Windows_NT' ">
     <Compile Include="**\*.wpf.cs" />
     <Compile Include="**\*.wpf.*.cs" />
-    <PackageReference Include="Xamarin.Forms.Platform.WPF" Version="5.0.0.2125" />
+    <PackageReference Include="Xamarin.Forms.Platform.WPF" Version="5.0.0.2291" />
```

Comment:
Let's not update this NuGet Package unless we have to, because it'll force all Xamarin.CommunityToolkit users to then upgrade to the latest version Xamarin.Forms